travis_sphinx/deploy.py

```diff
@@ -46,10 +46,11 @@ def deploy(ctx, branches, cname, message):
     outdir = ctx.obj['outdir']
     if token is None:
         click.ClickException("ERROR: GH_TOKEN is missing!")
+    github_host = os.environ.get('GH_HOST', 'github.com')
 
     branches = branches.split(',')
     if (branch in branches and (pr == 'false')) or tag:
-        remote = 'https://%s@github.com/%s.git' % (token, repo)
+        remote = 'https://%s@%s/%s.git' % (token, github_host, repo)
```

Comment:
@exp0nge Thank you for this contrib. Nevertheless could you use a clik option to pass the paramter to the deploy command in place of a environment variable.  (You can configure the option to look for an env var...http://click.pocoo.org/5/options/#values-from-environment-variables). In this way the option can be given as an args or an env var.

```python
...
@click.option(
    '-p', '--deploy-host,
     envvar='DEPLOY_HOST',
    default='github.com',
    show_default=True,
    )
...
@click.pass_context
def deploy(ctx, branches, cname, message, deploy_host):
    ...
    remote = https://%s@%s/%s.git' % (token, deploy_host, repo)

```

In the same time it would be nice to add a test for this usage. (a variation of https://github.com/exp0nge/travis-sphinx/blob/edcab3c9c1503cfea2883929a5c991aef04dc14a/tests/test_commands.py#L73)